config/runner.json

```diff
@@ -0,0 +1,11 @@
+{
+  "suites": {
+    "ecdsa-rdfc-2019": {"tags": ["ecdsa-rdfc-2019"]},
+    "ecdsa-sd-2023": {
+      "tags": ["ecdsa-sd-2023"],
+      "vcHolder": {
```

Comment:
Is this a subset of tests within `ecdsa-sd-2023`?

Reply:
@BigBlueHat the scenario I'm trying to use this for is running the tests locally on (a) just my implementation, (b) on just a subset of tests, e.g., `ecdsa-sd-2023`. This would let developers get up to speed before deploying to an external server, and specialize to specific cryptosuites.

Reply:
yes https://github.com/w3c/vc-di-ecdsa-test-suite/blob/965ee2e3e900be3044f873d5ce33982a6e42489a/tests/60-sd-interop.js#L76-L91

it's in the interop side of things. A future PR should probably roll the environment variable settings into a single json config file with defaultHolder in the vcHolder section.